### PR TITLE
Feature/updated frontend readme with project info

### PR DESCRIPTION
**Pull request by tthompson2**
Detailed description of project was added to the readme. The Readme includes information about all of the features that are expected to be delivered including the Notion documentation. Typos and spelling inconsistences were also edited.

### CODE REVIEW

**Comment by tthompson2** [comment]
[Update Frontend Readme with project information](https://trello.com/c/KPSNU9jy/53-update-frontend-readme-with-project-information)

File: README.md [hunk 1]
```diff
@@ -1,29 +1,31 @@
-# express-groomer
+# Express-Groomer
 
-- Welcome to your `Basic Single Page Application Repository`. Use this to start your own Greenfield Project using ReactJS and common industry standards.
+- Express Groomer connects groomers and clients.  The connection allows for groomers to meet at the clients residence or to meet at the groomers residence.  Clients have the ability to safely schedule an appointment and groomers can grow their business. Clients have the ability to search for groomers in their based on particular criteria. Groomers can manage appointment requests from users. Groomers will be able to schedule appointments. All of the features of the application are designed to empower both clients and groomers. The former will be empowered to search out for great care for their animals, and the ladder will be empowered meet wonderful clients and their animals for great pay.
```
Comment (cdlafortune) [inline README.md:4]: this looks good, thank you

File: src/components/pages/Landing/landingPageStyledComponents.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
+import styled from 'styled-components';
+
```
Comment (cdlafortune) [inline src/components/pages/Landing/landingPageStyledComponents.js:2]: should these changes outside the readme go with a different PR?

**Review by Zolw95 — APPROVED** [review]
The home/marketing page looks good. I would look for an ideal font size for /home and use the same font size for text. Maybe add some padding on the right and left to the paragraph under the map so it adds a little spacing to the right and left of the paragraph - making it's easier to read. Also, maybe try to do flex-direction column + align items center on a smaller screen for the paragraph and pet image above the map. Let's discuss if needed - Thanks!

**Review by jarrod847 — APPROVED** [review]
Good job Trevor!

File: src/components/pages/Landing/landingPageStyledComponents.js (hunk 2 above)
Comment (tthompson2) [reply src/components/pages/Landing/landingPageStyledComponents.js:2]: They should! I was a little bit confused about how to push that code. It came in when I did a pull request to make sure that the branched stayed current. I need to go about removing all of the information from this commit not related to the updated readme.
